src/MediatR/Mediator.cs

```diff
@@ -21,17 +21,32 @@ public class Mediator : IMediator
         private readonly ConcurrentDictionary<Type, Type> _genericHandlerCache;
         private readonly ConcurrentDictionary<Type, Type> _wrapperHandlerCache;
 
+        private PublishAsyncOptions _publishOption;
+
         /// <summary>
         /// Initializes a new instance of the <see cref="Mediator"/> class.
         /// </summary>
         /// <param name="singleInstanceFactory">The single instance factory.</param>
         /// <param name="multiInstanceFactory">The multi instance factory.</param>
-        public Mediator(SingleInstanceFactory singleInstanceFactory, MultiInstanceFactory multiInstanceFactory)
+        public Mediator(SingleInstanceFactory singleInstanceFactory, MultiInstanceFactory multiInstanceFactory) :
+            this(singleInstanceFactory, multiInstanceFactory, new MediatorConfiguration())

```

Comment:
I'd make it more readable: `new MediatorConfiguration { PublishAsyncOptions = PublishAsyncOptions.Parallel  }`, otherwise reader needs to go to the class's definition and learn the behavior from there